android/src/main/java/com/hathway/mparticle/capacitor/MParticleCapacitor.java

```diff
@@ -70,7 +70,7 @@ public Product createMParticleProduct(JSObject productData) {
         return new Product.Builder(
             (String) productData.getString("name"),
             (String) productData.getString("sku"),
-            (double) productData.getInteger("cost"))
```

Comment:
@dghathway This actually was changed because it was causing a bug where the cents on the price was getting truncated.  This should be an improvement, but may be unexpected one to consider.